src/iarc7_motion/iarc_tasks/track_roomba_task.py

```diff
@@ -153,13 +156,11 @@ def get_desired_command(self):
                 if not self._check_max_roomba_range():
                     return (TaskAborted(msg='The provided roomba is not found or not close enough to the quad'),)
 
-                if self._check_max_ending_roomba_range():
-                    return (TaskDone(),)
-
                 roomba_x_velocity = self._roomba_odometry.twist.twist.linear.x
                 roomba_y_velocity = self._roomba_odometry.twist.twist.linear.y
                 roomba_velocity = math.sqrt(roomba_x_velocity**2 + roomba_y_velocity**2)
 
+                # The overshoot is back in town
```

Comment:
Update this comment to explain what is going on (i.e. using the direction the roomba is moving)